### DISCUSSION TITLE
blog/how-to-set-commit-msg-with-husky

### DISCUSSION THREAD

**Discussion by giscus** [jhsung23/my-site#5] [Comments]
# blog/how-to-set-commit-msg-with-husky

혼자 어긋나는 커밋 작성하지 않도록 무의식을 통제해 봅시다…

https://sungjihyun.vercel.app/blog/how-to-set-commit-msg-with-husky

**Comment by kimpra2989** [comment]
블로그에 좋은 글이 많네요. 공유 감사합니다!
